test/end-to-end/pages/applicant/spousenotapplyingreason.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function (answer) {
+module.exports = async function (language = 'en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
 
```

Comment:
Shouldn't we also check for some text on this page or are we happy just to check the button text?

Reply:
done.